### PR TITLE
Updated Quota limits for new plans

### PR DESCRIPTION
**Pull request by Nichelle2571**
Update only with new plans 8/15

### CODE REVIEW

File: articles/faq/daily-and-monthly-quota-limits.md [hunk 1]
```diff
@@ -11,14 +11,13 @@ You can view your current quota limits on the [Account Usage](http://app.sparkp
...
-* Free plans that have been validated with a credit card are limited to a maximum of 20,000 messages per day. If you need a higher daily limit, upgrade to a paid plan on the [Billing](https://app.sparkpost.com/account/billing) page.
-* For paid plans, you can request an increase to your daily limit on the [Account Usage](http://app.sparkpost.com/account/usage) page.
+* For free plans, the daily message allowance is limited to 750 messages per day once you’ve verified your email address. Accounts without a verified email address are limited to 100 messages per day. Be sure you verify the email address used to create your account by clicking the link in the welcome email—this will ensure you are eligible for the maximum sending limit.
+* If you need a higher daily limit, upgrade to a paid plan on the [Billing](https://app.sparkpost.com/account/billing) page.
+* For paid plans, you can request an increase to your daily limit on the [Account Usage](http://app.sparkpost.com/account/usage) page. All limit requests are reviewed based upon your accounts sending performance and history with SparkPost.
```
Comment (richleland) [inline articles/faq/daily-and-monthly-quota-limits.md]: accounts should be account's

Comment (Nichelle2571) [inline articles/faq/daily-and-monthly-quota-limits.md]: I think it should be accounts. Account's says that the account owns something.

Comment (richleland) [inline articles/faq/daily-and-monthly-quota-limits.md]: it does - it possesses the sending performance and history

**Comment by Nichelle2571** [comment]
Made the change

File: articles/faq/daily-and-monthly-quota-limits.md (hunk 1 above)
Comment (richleland) [inline articles/faq/daily-and-monthly-quota-limits.md]: lol wrong accounts - I meant in the last sentence - sorry!!

**Comment by Nichelle2571** [comment]
LOL. Oh OK.

**Pull request merged**